IGC/VectorCompiler/lib/GenXCodeGen/GenXCoalescing.cpp

```diff
@@ -203,6 +202,8 @@ SPDX-License-Identifier: MIT
 #include <map>
 #include <vector>
 
+#define DEBUG_TYPE "GENX_COALESCING"
+
```

Comment:
What's this for? I see reference here: https://llvm.org/docs/ProgrammersManual.html, but I'm not sure why it's needed here for LLVM 14 compatibility.

Reply:
> What's this for? I see reference here: https://llvm.org/docs/ProgrammersManual.html, but I'm not sure why it's needed here for LLVM 14 compatibility.

There are some headers in the `#include` that has `#undef DEBUG_TYPE`. They were added in LLVM 14 for some reason.